tests/src/test_cases/widgets/test_msgbox.c

```diff
@@ -144,4 +144,38 @@ void test_msgbox_close_async_modal(void)
     TEST_ASSERT_NOT_NULL(msgbox);
 }
 
+void test_msgbox_content_auto_height(void)
+{
+    // If parent is NULL the message box will be modal
+    msgbox = lv_msgbox_create(NULL);
+    lv_msgbox_add_title(msgbox, "The title");
+    lv_msgbox_add_text(msgbox, "The text");
+    lv_msgbox_add_footer_button(msgbox, "Apply");
+    lv_msgbox_add_footer_button(msgbox, "Close");
+    lv_msgbox_add_header_button(msgbox, LV_SYMBOL_AUDIO);
+    lv_msgbox_add_close_button(msgbox);
+
+    /*change size of msgbox*/
+    lv_obj_set_size(msgbox, lv_pct(80), lv_pct(80));
+
+    lv_obj_update_layout(msgbox);
+    lv_obj_t * header = lv_msgbox_get_header(msgbox);
+    lv_obj_t * footer = lv_msgbox_get_footer(msgbox);
+    lv_obj_t * content = lv_msgbox_get_content(msgbox);
+
+    int32_t h_header = (header == NULL) ? 0 : lv_obj_get_height(header);
+    int32_t h_footer = (footer == NULL) ? 0 : lv_obj_get_height(footer);
+    int32_t h_content = lv_obj_get_height(content);
+
+    int32_t h_obj_content = lv_obj_get_content_height(msgbox);
+    int32_t h_msgbox_element_sum  = h_header + h_footer + h_content;
+
+    TEST_ASSERT_NOT_NULL(msgbox);
+
+    TEST_ASSERT_EQUAL(h_obj_content, h_msgbox_element_sum);
+
+    // Since msgbox has no parent, it won´t be clean up at tearDown()
```

Comment:
```suggestion
    /* Since msgbox has no parent, it won´t be clean up at tearDown() */
```

Reply:
![image](https://github.com/lvgl/lvgl/assets/15370479/b3428a10-a0af-4d79-8dde-7aac82f4cb44)

Now , support  both  msgbox's default size (by LV_SIZE_CONTENT)  and  msgbox' manual size.

However, for `test_msgbox.c` in the `test_cases` folder, 
when I tested whether the msgbox's height is set to LV_SIZE_CONTENT or not, 
it's OK. 

![image](https://github.com/lvgl/lvgl/assets/15370479/1f07ca73-b9c5-4234-855d-567a8af20328)

But when the command 
```
lv_obj_update_content(msgbox); 
```
is added, an error occurs. 

![image](https://github.com/lvgl/lvgl/assets/15370479/236a5640-a798-46da-a772-99f8d88675cf)

I don't know what caused the error in `test_msgbox.c`.